python-mini-projects/projects/Dns_record/dns_record.py

```diff
@@ -0,0 +1,29 @@
+#Simple program to fetch dns record of a given website
+
+import dns.resolver
+
+#Dictionary to store the dns record of a website
+dns_record = {}
+
+#User defined website
+website = input("Enter the name of the website: ")
+
+#Fetching the 'A' record of the website and storing it in the dictionary
+a_record = dns.resolver.resolve(website, 'A')
+for ipval in a_record:
+    dns_record['A_Record_IP'] = ipval.to_text()
+
+#List to store the mx records of a website
+mx_record_list = []
+
+#Fetching the mx records and storing them in the dictionary
+mx_record = dns.resolver.resolve(website,'MX')
+for server in mx_record:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)